### PR TITLE
Add country to info call #403

### PR DESCRIPTION
**Pull request by tobifaf**
See #403

### CODE REVIEW

File: frontend/src/api/login.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface RequestRegisterChain {
   name: string;
   description: string;
   address: string;
+  country_code: string;
```
Comment (lil5) [inline frontend/src/api/login.ts:19]: Country should be considered write only. 
So the country code should be able to be set from the api but not returned (except in the info route)

File: frontend/src/components/Counters/Counters.tsx [hunk 2]
```diff
@@ -48,7 +48,11 @@ export default function Counters() {
 
         <div className="">
           <div className="h-20 text-6xl font-serif text-stroke-base-100">
-            12
+            {isVisible ? (
+              <SingleCounter end={info?.total_countries || 0} step={193} />
```
Comment (lil5) [inline frontend/src/components/Counters/Counters.tsx:52]: Set the step to 1

**Review by lil5 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Counters/Counters.tsx (hunk 2 above)
Comment (tobifaf) [reply frontend/src/components/Counters/Counters.tsx]: fixed

File: frontend/src/api/login.ts (hunk 1 above)
Comment (tobifaf) [reply frontend/src/api/login.ts:19]: fixed in 54e6f346223f0e170ae1a7ff22c07a198837f07f

**Pull request merged**